packages/controls/src/controls/style/schema.ts

```diff
@@ -0,0 +1,215 @@
+import { z } from 'zod'
```

Comment:
Converted types from https://github.com/makeswift/makeswift/pull/781/files#diff-fe2757ce8e16ddd867978a37261e04fe23962f06fdaafd490fc0b26f226411ce and https://github.com/makeswift/makeswift/tree/main/packages/runtime/src/css to Zod schemas.